cypress/support/e2e.ts

```diff
@@ -37,9 +37,15 @@ declare global {
        */
       userState?: Partial<UserState>
     }
+    interface Chainable {
+      /** Get one or more DOM elements by data-testid attribute. */
+      getByTestId<E extends Node = HTMLElement>(selectorId: string): Chainable<JQuery<E>>
+    }
```

Comment:
it feels like this should be done by the command

Reply:
Cypress requires that you type and add a command separately, as is done here.